Chart.yaml

```diff
@@ -15,12 +16,12 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 4.0.0
+version: 4.1.0
```

Comment:
@renchap Does your commend on my PR also apply here? https://github.com/mastodon/chart/pull/5#issuecomment-1440044699 As in, you as the maintainers will increase it when releasing?

Reply:
Please update the chart version soon, I don't mind who does it but I'm also waiting for my changes from #47 to be available in a versioned relase. Latest release v4.0.0 is already ~3 month old.

Reply:
I updated this PR to update to mastodon 4.1.1 in the meantime

Reply:
I updated this PR to update to mastodon 4.1.2